tests/integration/test_format_txts.py

```
@@ -0,0 +1,334 @@
+    # Verify that a file was found
+    assert result is not None
+
+    # Read the content to check if it's merged
+    content = result.read_text()
+    assert "Phone message 1" in content
+    assert "Phone message 2" in content
+    assert "Email message 1" in content
+    assert "Email message 2" in content
+
+
+@pytest.mark.parametrize("date, end_date", [("2024-01-01", "2024-02-01"), (None, None)])
```

Ruff (0.14.2)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)